msg/AutoTrimStatus.msg

```diff
@@ -0,0 +1,17 @@
+uint64 timestamp             # Time since system start (microseconds)
```

Comment:
1. Can you add first line comment explaining what this does  that then appears in the docs as an [overview](https://docs.px4.io/main/en/msg_docs/) (and is convenient anyway when reading the source)
2. Might be nice to document all these other ones as well, if only to note that they are internal. 

For the comment, something like:

```suggestion
# Fixed-wing auto trim calibration state and current estimates

uint64 timestamp             # Time since system start (microseconds)
```